.github/workflows/push.yml

```diff
@@ -146,12 +151,13 @@ jobs:
         run: cargo sweep --maxsize ${{ env.MAX_TARGET_SIZE }}
 
   docs:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Why change to latest?

Reply:
22.04 is the [latest](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)

Reply:
It is today, in a bit over half a year it will not be. Please use specific versions to avoid surprises in the future.